torch_xla/utils/gcsfs.py

```diff
@@ -0,0 +1,180 @@
+# To use this module the Google Cloud Storage libraries need to be installed:
+#
+#  pip install --upgrade google-cloud-storage
+#
+# For information on how to setup authentication follow the link below:
+#
+#  https://cloud.google.com/storage/docs/reference/libraries
+#
+
+import collections
+import io
+import os
+import re
+import tempfile
+from google.cloud import storage as gcs
```

Comment:
Maybe we could wrap this in a `try...catch` block and give an error message with `pip install --upgrade google-cloud-storage`?

Reply:
Good idea!